27-06-2018:14:26:02 INFO     [simple_runtime.py:299] image 9c7545cf-8d2d-4e18-af3c-dc4f99e39491: Creating image
27-06-2018:14:26:02 INFO     [simple_runtime.py:396] image 9c7545cf-8d2d-4e18-af3c-dc4f99e39491: slice 0: Sending build request
27-06-2018:14:27:06 INFO     [simple_runtime.py:405] image 9c7545cf-8d2d-4e18-af3c-dc4f99e39491: slice 0: Build request sent
Traceback (most recent call last):
  File "pkg/com_bonseyes_base/bin/be-admin", line 43, in <module>
    main.main()
  File "/home/z638420/wp3-project-objectdetection2/pkg/com_bonseyes_base/lib/cli/__main__.py", line 63, in main
    args.func(args)
  File "/home/z638420/wp3-project-objectdetection2/pkg/com_bonseyes_base/lib/cli/commands/run.py", line 446, in <lambda>
    parser.set_defaults(func=lambda args: run(root_parser, parser, args))
  File "/home/z638420/wp3-project-objectdetection2/pkg/com_bonseyes_base/lib/cli/commands/run.py", line 335, in run
    execution.execute()
  File "/home/z638420/wp3-project-objectdetection2/pkg/com_bonseyes_base/lib/impl/executor/simple_executor.py", line 793, in execute
    worker.create_instance()
  File "/home/z638420/wp3-project-objectdetection2/pkg/com_bonseyes_base/lib/impl/executor/simple_executor.py", line 219, in create_instance
    image.build()
  File "/home/z638420/wp3-project-objectdetection2/pkg/com_bonseyes_base/lib/impl/runtime/simple_runtime.py", line 420, in build
    + str(data['errorDetail']) + "\n" + log_info)
Exception: image 9c7545cf-8d2d-4e18-af3c-dc4f99e39491: Error while building:
 {'code': 100, 'message': "The command '/bin/sh -c apt-get update &&     apt-get install -y --no-install-recommends             python3 python3-pip python3-setuptools python3-wheel &&     apt-get clean && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100"}
Step 1/4 : FROM ubuntu:xenial
 ---> 5e8b97a2a082
Step 2/4 : RUN apt-get update &&     apt-get install -y --no-install-recommends             python3 python3-pip python3-setuptools python3-wheel &&     apt-get clean && rm -rf /var/lib/apt/lists/*
 ---> Running in d16a6b03abce
Err:1 http://archive.ubuntu.com/ubuntu xenial InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:2 http://security.ubuntu.com/ubuntu xenial-security InRelease
  Temporary failure resolving 'security.ubuntu.com'
Err:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:4 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Reading package lists...
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-updates/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/xenial-backports/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/xenial-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package python3
E: Unable to locate package python3-pip
E: Unable to locate package python3-setuptools
E: Unable to locate package python3-wheel

